site/src/pages/UserSettingsPage/OAuth2ProviderPage/OAuth2ProviderPageView.tsx

```diff
@@ -0,0 +1,94 @@
+import Button from "@mui/material/Button";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import { type FC } from "react";
+import type * as TypesGen from "api/typesGenerated";
+import { AvatarData } from "components/AvatarData/AvatarData";
+import { Avatar } from "components/Avatar/Avatar";
+import { ErrorAlert } from "components/Alert/ErrorAlert";
+import { TableLoader } from "components/TableLoader/TableLoader";
+
+export type OAuth2ProviderPageViewProps = {
+  isLoading: boolean;
+  error?: unknown;
```

Comment:
Just realized something – do we want to make `error` required, so that we don't forget to pass it into the view? Hopefully it's `undefined` 99% of the time, but I could see someone forgetting to pass something down

Reply:
Good call!  Will fix.